File: JavaScript/beginner_problems_solutions/1010.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var code1 = parseInt(lines[0].split(' ')[0]);
var quantity1 = parseInt(lines[0].split(' ')[1]);
var price1 = parseFloat(lines[0].split(' ')[2]);

var code2 = parseInt(lines[1].split(' ')[0]);
var quantity2 = parseInt(lines[1].split(' ')[1]);
var price2 = parseFloat(lines[1].split(' ')[2]);

var total = (quantity1 * price1) + (quantity2 * price2);

console.log("VALOR A PAGAR: R$ " + total.toFixed(2));

File: Python/beginner_problems_solutions/1010.py
product1 = input().split()
code1, quantity1, price1 = product1
product2 = input().split()
code2, quantity2, price2 = product2
total_price1 = int(quantity1) * float(price1)
total_price2 = int(quantity2) * float(price2)
final_price = total_price1 + total_price2
print(f"VALOR A PAGAR: R$ {final_price:.2f}")